esp/services/ws_dfu/ws_dfuHelpers.cpp

```diff
@@ -152,7 +152,7 @@ bool WsDFUHelpers::addToLogicalFileList(IPropertyTree& file, const char* nodeGro
                 lFile->setKeyType("Distributed");
         }
 
-        bool isFileCompressed = file.getPropBool(getDFUQResultFieldName(DFUQResultField::iscompressed));
+        bool isFileCompressed = file.getPropBool(getDFUQResultFieldName(DFUQResultField::iscompressed)) || file.getPropBool(getDFUQResultFieldName(DFUQResultField::blockCompressed));
```

Comment:
I'd thought that the `LogicalFileResolver::processFile` function was involved with serializing the file info via ESP's (eventual) call to `CDistributedFileDirectory::getDFAttributesFilteredIterator()` If true, that would mean that `@compressed` is set to true when _either_ `@blockCompressed` or `@rowCompressed` is true, in which case we shouldn't need to check here for either of those explicitly. 

However, if I'm wrong, which it appears I am, then don't we also need to check for `@rowCompressed` here? Furthermore, then how did the check for just `@compressed` ever work as expected?